kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/CloudServices/CloudServiceConfigurationsUi.java

```diff
@@ -170,42 +139,42 @@ public void onFailure(Throwable ex) {
             }
 
             @Override
-            public void onSuccess(final List<String> pids) {
+            public void onSuccess(List<String> result) {
+                final List<String> pidsResult = result;
 
                 CloudServiceConfigurationsUi.this.gwtXSRFService
                         .generateSecurityToken(new AsyncCallback<GwtXSRFToken>() {
 
-                    @Override
-                    public void onFailure(Throwable ex) {
-                        FailureHandler.handle(ex, EntryClassUi.class.getName());
-                    }
-
-                    @Override
-                    public void onSuccess(GwtXSRFToken token) {
-                        CloudServiceConfigurationsUi.this.gwtComponentService.findFilteredComponentConfigurations(token,
-                                new AsyncCallback<List<GwtConfigComponent>>() {
-
                             @Override
                             public void onFailure(Throwable ex) {
-                                logger.log(Level.SEVERE, ex.getMessage(), ex);
                                 FailureHandler.handle(ex, EntryClassUi.class.getName());
                             }
 
                             @Override
-                            public void onSuccess(List<GwtConfigComponent> result) {
```

Comment:
I still don't understand it. The order of stack PIDs it not changed. Maybe you can point me the code where you think the issue is.